src/dhali/transaction_utils.py

```diff
@@ -143,6 +144,7 @@ def _validation(
     estimated_payment_claim_doc_ref = db.collection(root_private_collection_name).document(root_private_payment_claim_doc.id).collection(estimate_collection_name).document(uuid_estimate)
     estimated_payment_claim_doc_ref.set(
         {
+            "timestamp": datetime.datetime.now(),
```

Comment:
I think we want this to be utc, like in [the docs](https://firebase.google.com/docs/reference/js/v8/firebase.firestore.Timestamp).  To guarantee utc, I think we use [utcnow](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow).

Reply:
Good catch!